pyproject.toml

```diff
@@ -37,6 +37,7 @@ issue-tracker = "https://github.com/fbruzzesi/timebasedcv/issues"
 
 [project.optional-dependencies]
 polars = ["polars"]
+pandas = ["pandas"]
```

Comment:
Are there min versions required by narwhals?

Reply:
thanks!

I'll test this backwards and see how far we can get, but `.filter` I'd expect should go back essentially forever

Reply:
would pandas>=1.2.0 and Polars>=0.20.3 work for you? I've set those as the minimum, and have tested with the minimum versions

Reply:
LGTM